CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Removed
 
```

Comment:
I think the switch to an inventory file needs to be mentioned in the changelog explicitly. It does change functionality for end users, in that pinning to a buildpack version now affects the versions available for use etc.

Reply:
Pinning to exact specific versions, yeah. Getting new versions when pinning to a major already required a buildpack update. I will add this! :)